code/__DEFINES/atmospherics.dm

```diff
@@ -495,14 +499,29 @@ GLOBAL_LIST_INIT(pipe_paint_colors, sortList(list(
 		"cyan" = rgb(0,255,249),
 		"dark" = rgb(69,69,69),
 		"green" = rgb(30,255,0),
-		"grey" = rgb(255,255,255),
+		"grey" = rgb(254,254,254),
```

Comment:
These *have* to become defines. They're referenced more or less everywhere, meaning wanting to change the color is a way bigger pain in the ass than it has any right to be, like how this specific value of 254, 254, 254 crops up a lot.